framework/serialize.cpp

```diff
@@ -284,13 +312,11 @@ static bool readDir(const UString &name, std::map<UString, std::string> &content
 	return true;
 }
 
-static bool writePack(const UString &name, const std::map<UString, std::string> &contents)
+static bool writePack(const UString &path, const std::map<UString, std::string> &contents)
 {
 	mz_zip_archive archive;
 	memset(&archive, 0, sizeof(archive));
-
-	auto path = name + ".zip";
-
```

Comment:
I need quick way of telling if file is saved game, so user do not have to wait for save screen, so I am detecting saves by .save extension.